docker-compose/scripts/run.sh

```diff
@@ -27,4 +27,8 @@ fi
 
 log "Starting Kuzzle..."
 
-exec ./bin/kuzzle "$@"
+if [ -n "$KUZZLE_PLUGINS" ]; then
+  enable_plugins="--enable-plugins $KUZZLE_PLUGINS"
+fi
```

Comment:
Needs to be included only if `kuzzle start` is called.

i.e.
```
docker run -e KUZZLE_PLUGINS=foo kuzzleio/kuzzle kuzzle createFirstAdmin 
```
will run `bin/kuzzle createFirstAdmin --enable-plugins foo`